test/k8s-integration/main.go

```diff
@@ -331,6 +332,7 @@ func handle() error {
 		if err != nil {
 			return fmt.Errorf("failed to prepull images: %s, err: %v", out, err)
 		}
+		time.Sleep(15 * time.Minute)
```

Comment:
Is there a more deterministic way to detect when the prepull is complete? Is the script blocking?

Reply:
the script creates a daemonSet with desired image and the pods will download the container image. Similar to driver install and other windows tests, we currently just wait for sometime. We can improve this part by waiting for pod state changes next step.

Reply:
Please open an issue so we can track followup items